gordon/main.py

```diff
@@ -40,6 +41,8 @@
 import ulogger
 
 from gordon import __version__ as version
```

Comment:
Should this be imported less directly (per the style guide?

Reply:
While this isn't a part of this PR, importing meta stuff from `__init__` is a special case IMO. `import gordon; gordon.__version__` would work, but could break if we do anything more fancy with `gordon.__init__` (e.g. similar to how we use `gordon_gcp.__init__` and `gordon_janitor_gcp.__init__`, not that we would) since it'd most likely introduce a circular dependency.